docs/api/world.md

```diff
@@ -1,175 +1,358 @@
 # World
+A World contains entities which have components. The World is queryable and can be used to get entities with a specific set of components and to perform different kinds of operations on them.
 
-A World contains entities which have components. The World is queryable and can be used to get entities with a specific set of components.
+## Functions
+### new
+Creates a new world (necessary step to utilize the methods of the class)
 
-# Functions
-
-## new()
+`World` utilizes a class, meaning JECS allows you to create multiple worlds.
 ```luau
 function World.new(): World
 ```
-Creates a new world.
 
 Example:
-::: code-group
 
+::: code-group
 ```luau [luau]
 local world = jecs.World.new()
+local myOtherWorld = jecs.World.new()
 ```
 
 ```ts [typescript]
 import { World } from "@rbxts/jecs";
 
 const world = new World();
+const myOtherWorld = new World();
 ```
-
 :::
 
-## entity()
+## Methods

```

Comment:
Methods should be h1 `#` and every individual function should be `##`. Otherwise the individual functions wont be indexable from the page.